incentivizing_rewarding_participants.adoc

```diff
@@ -1,87 +1,157 @@
+In this way, receiving items that allow them to publicly express their affiliation with an important peer group—or signal their status _within_ that group—might motivate them to contribute to a project.
+
+This reward model often requires substantial resources, however. The work of packing and shipping branded items to members of a global community can be time-consuming.
+Moreover, production and distribution of branded materials can involve significant monetary expenses.
+
+=== Community spotlights and awards
+
+Community spotlights and awards help you showcase the people who make your community great.
+Some open source communities publish occasional blog posts or run video series to spotlight contributors who have made significant impacts on the project.
+For instance, they might feature a monthly "Meet the Contributors" or "Community Spotlight" series that highlights key contributors, allowing them to tell the community more about themselves.
+Other communities might develop an annual award series, recognizing a "Contributor of the Year" or "Most Valuable Maintainer."
+The project might then feature award winners at events or in featured interviews posted online.
+
+Because this reward model involves opportunities for gaining visibility, it's useful when you know participants are motivated by a desire to enhance their personal reputations and grow their professional networks.
+The model is also particularly useful for attracting and retaining diverse contributors.
+Communities spotlighting contributors from under-represented groups provide a way for new, potential contributors from those groups to "see themselves" in their projects and feel more confident joinging them.
+Moreover, when communities allow _members themselves_ to nominate, vote for, and honor their peers, they reinforce communal values and strengthen social bonds.
+So implementing this reward model can also help you connect with contributors who see community participation as a way to form bonds with others, to feel a sense of group belonging, and to feel connected to a purpose.
+
+Note, however, that choosing to showcase certain contributors creates the impression that those contributors embody the principles and enact the behaviors your community finds most valuable—and that more contributors should aspire to emulate them.
+Additionally, communities that implement this reward model might be accused of "playing favorites" or creating an insular culture.
+To mitigate this, you may wish to formalize and publish a basic procedure by which your community selects members to spotlight or receive awards.
+The procedure might involve, community members nominating one another to be featured by writing testimonials about their peers' impacts on a project or the way they feel those peers embody the project's guiding values.
+The procedure might also incorporate an opportunity for nominated individuals to *decline* those nominations if they'd prefer to avoid the spotlight.
```

Comment:
I think the *emphasis* here isn't needed, just being mindful of not overusing emphasis across the chapter.

Reply:
Agree. Thanks.